tests/unit/metadata/test_multi_table.py

```diff
@@ -50,25 +111,39 @@ def test__validate_missing_relationship_keys_foreign_key(self):
         """
         # Setup
         parent_table = Mock()
-        parent_table._primary_key = 'users_id'
+        parent_table._primary_key = 'id'
         parent_table._columns = {
-            'users_id': {'sdtype': 'numerical'},
+            'id': {'sdtype': 'numerical'},
             'session': {'sdtype': 'numerical'},
             'transactions': {'sdtype': 'numerical'},
         }
         parent_table_name = 'users'
-        parent_primary_key = 'users_id'
+        parent_primary_key = 'id'
+
+        child_table = Mock()
+        child_table._primary_key = 'session_id'
+        child_table._columns = {
+            'user_id': {'sdtype': 'numerical'},
+            'session_id': {'sdtype': 'numerical'},
+            'transactions': {'sdtype': 'numerical'},
+        }
         child_table_name = 'sessions'
-        child_foreign_key = 'session_id'
+        child_foreign_key = 'id'
```

Comment:
Is the description of this test case wrong? The child foreign key seems to be in the parent_table primary keys.

Reply:
Yyes the description maybe wrong I need to revisit them!